### PR TITLE
[UI] Multi Select

### PR DESCRIPTION
**Pull request by adityarb88**
Fixes: #576 

Added styles for `<select multiple>`

image

Form control in a grid:
image

Multi Select with focus:
image

Multi Select without focus:
image

Note: Some browsers may have different styles for scroll bar and highlight

Tested on: Chrome, Firefox, Safari, IE 10, 11, Edge

Signed-off-by: Aditya Bhandari <adityab@vmware.com>

### CODE REVIEW

File: src/clarity-angular/forms/_select.clarity.scss [hunk 1]
```diff
@@ -73,5 +72,20 @@
                 border-color: $clr-form-field-border-color;
             }
         }
+
+        //Multiple Select
+
+        &.multiple select[multiple] {
```
Comment (mathisscott) [inline src/clarity-angular/forms/_select.clarity.scss]: Why do we need an extra class on the `.select` wrapper?

It doesn't seem necessary. Can't we just say `select[multiple]` here?

Comment (adityarb88) [inline src/clarity-angular/forms/_select.clarity.scss]: Updated

File: src/app/selects/selects.demo.html [hunk 2]
```diff
@@ -46,6 +46,34 @@
             </div>
         </div>
     </section>
+
+    <section class="form-block">
+        <label>Multi Select</label>
+        <div class="form-group">
+            <label for="selects_1">Multi Select</label>
+            <div class="select multiple">
```
Comment (mathisscott) [inline src/app/selects/selects.demo.html]: Are we still putting `.multiple` on the demos?

File: src/clarity-angular/forms/_select.clarity.scss [hunk 3]
```diff
@@ -73,5 +72,29 @@
                 border-color: $clr-form-field-border-color;
             }
         }
+
+        //Multiple Select
+        &.multiple::after {
```
Comment (mathisscott) [inline src/clarity-angular/forms/_select.clarity.scss]: Oh, I see. We have to put `content:none` after the wrapper block...

File: src/clarity-angular/forms/_select.clarity.scss [hunk 4]
```diff
@@ -73,5 +72,29 @@
...
+            content: none;
+        }
+    }
+
+    //Multiple Select
+    select[multiple] {
```
Comment (mathisscott) [inline src/clarity-angular/forms/_select.clarity.scss]: Thanks. This looks a lot cleaner to me...

**Pull request merged**